.travis.yml

```diff
@@ -46,6 +46,13 @@ jobs:
     - name: storybook-smoke-test (node@production)
       <<: *node-prod
       script: yarn storybook-smoke-test
+    - name: yarn serve (node@production)
+      <<: *node-prod
+      script:
+        # `yarn build` takes some time so let's simulate it instead.
+        - mkdir build && touch build/index.html
+        - yarn serve &
```

Comment:
When I run this locally with the commit in this branch that causes a TS error, I don't see any errors. Do you know why that is? I'm on Node 10.15.0. Why does it cause an error on Travis?

Reply:
Do you run with `&` or without?

Travis runs Node v10.15.3. I definitely get the error locally (running `yarn serve` only)

Reply:
I am on v10.13.0 locally (and v11, sometimes)

Reply:
Hmm. I upgraded to 10.15.3 locally and I still don't see any errors by running:

```
touch build/index.html
yarn serve
```

It doesn't seem to make a difference if I send `yarn serve` to the background or not.

Reply:
Try removing your `node_modules` and getting fresh deps?

Reply:
Whoops! I was on the "revert revert" commit, not the "revert" commit 😎  I can reproduce the error locally now.